.github/workflows/update_pyapi_stubs.yml

```diff
@@ -0,0 +1,55 @@
+name: Update Python API Stubs
+
+permissions:
+  contents: write
+  pull-requests: write
+
+on:
+  schedule:
+  - cron: "0 2 * * 1-5"
+  workflow_dispatch:  # Allow manual triggering
+
+jobs:
+  update-stubs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
```

Comment:
Please use the commit hash as version:
e.g: @271a8d0340265f705b14b6d32b9829c1cb33d45e  # v7.0.8

Reply:
Done.